### DISCUSSION TITLE
Docs updates

### DISCUSSION THREAD

**Discussion by Michael21107** [scripthookvdotnet/scripthookvdotnet#1389] [Q&A] [answered]
if the only commits are for docs(api-v3) as here https://github.com/scripthookvdotnet/scripthookvdotnet-nightly/releases/tag/v3.6.0-nightly.44, is there a reason to update if im not a dev?

**Comment by kagikn** [comment]
The nightly release repo is targeted for both non-dev and dev people. You can ignore some of releases that don't have no important changes for you.

**Reply by Michael21107** [reply]
yea, im asking if docs updates have any effects for non-devs

**Reply by kagikn** [reply, marked as answer]
nah docs updates don't have any effects for non-devs. Commits that changes only some of docs doesn't have effect on what binaries will be produced, apart from something trivial for non-devs such as timestamps when they got built.

**Reply by Michael21107** [reply]
thanks
